app/helpers/question_helper.rb

```diff
@@ -0,0 +1,34 @@
+
+
+module QuestionHelper
+  DAY = 24*360
+
+  def format_asked_time(question)
+    formated_string = ""
+    Time.zone= "Eastern Time (US & Canada)"
+    now = Time.zone.local_to_utc(Time.now)
+
+    interval = (now-question.created_at).to_i
+
+    ##TODO add days
+    if interval > DAY
```

Comment:
Rails has a lot of this built in as part of active support. See time_ago_in_words on http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html